### ISSUE TITLE
OrbitPlotter2D plots orbit in fron of the body

### ISSUE THREAD

**Issue by shreyasbapat** [poliastro/poliastro#345]
I was just trying out the ``OrbitPlotter2D`` just introduced and I found that it shows terribly bad plots for ``poliastro.examples`` (or maybe more correctly, the objects very near to earth). 
🐞 **Problem**

Look at the plots of ``molniya`` and ``iss``. The line appears in front of the body and becomes difficult to see. It can be the extension of #278 

![screenshot from 2018-04-15 12-23-45](https://user-images.githubusercontent.com/24257914/38775772-f5e862d8-40a7-11e8-8c5c-f9bcb0c7273d.png)

🖥 **Please paste the output of following commands**

`conda info -a`

`python -c "import poliastro.testing; poliastro.testing.test()"`

🎯 **Goal**

It is important for one to be able to visualize the orbits using ``plotly``. As in case of ``OrbitPlotter`` , the orbits can still be recognized. 
💡 **Possible solutions**

We can probably change the colors of the orbit  (Only in case of examples ?) 
Or we may  follow the proposed solutions in #278 
📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

**Comment by Juanlu001** [comment]
A new feature was added to plotly that allows you to draw fixed size shapes with respect to the axes:

https://github.com/plotly/plotly.js/pull/2532

Can you try with the development version of plotly to see if it fixes the issue?

**Comment by shreyasbapat** [comment]
Yes! I will go through the PR to understand what they have done!

**Comment by Juanlu001** [comment]
I'm labeling this as a bug so we give it the attention it deserves.
